.pre-commit-config.yaml

```diff
@@ -7,11 +7,6 @@ repos:
       - id: end-of-file-fixer
       - id: requirements-txt-fixer
       - id: check-yaml
-  - repo: https://github.com/asottile/pyupgrade
-    rev: v3.15.0
-    hooks:
-    - id: pyupgrade
-      args: ["--py38-plus"]
```

Comment:
Any clues on why `pyupgrade` is failing on the precommit hooks? I couldn't make it to work even after several different configuration. 

Locally pyupgrade didn't re-write any files. 

Pinging @purificant